src/core.js

```diff
@@ -739,26 +742,29 @@ $.extend( $.validator, {
 			return undefined;
 		},
 
-		defaultMessage: function( element, method ) {
-			return this.findDefined(
-				this.customMessage( element.name, method ),
-				this.customDataMessage( element, method ),
-
-				// 'title' is never undefined, so handle empty string as undefined
-				!this.settings.ignoreTitle && element.title || undefined,
-				$.validator.messages[ method ],
-				"<strong>Warning: No message defined for " + element.name + "</strong>"
-			);
-		},
+		defaultMessage: function( element, rule ) {
```

Comment:
I don't see why change `message: this.defaultMessage( element, { method: method } )`

May add some kind of deprecation comment about the old parameter type so it could be removed on next major version